packages/blade/src/components/IconButton/StyledIconButton.native.tsx

```diff
@@ -0,0 +1,55 @@
+import type { ReactElement } from 'react';
+import { useState } from 'react';
+import styled from 'styled-components/native';
+
+import type { StyledIconButtonProps } from './StyledIconButton.d';
+import { makeAccessible } from '~utils';
+import type { ColorContrastTypes } from '~tokens/theme/theme';
+
+type GetIconColorTokenParams = {
+  isPressed: boolean;
+  contrast: ColorContrastTypes;
+};
+
+type State = 'active' | 'default';
+type IconColorToken = `feedback.neutral.action.icon.link.${State}.${ColorContrastTypes}Contrast`;
+
+const getIconColorToken = ({ isPressed, contrast }: GetIconColorTokenParams): IconColorToken => {
```

Comment:
not a blocker: inline it since it's used anywhere else?

Reply:
Resolved in https://github.com/razorpay/blade/pull/613/commits/1522ef438febf3bd4460334b869493793813b17d